apps/template/src/components/AppContent.tsx

```diff
@@ -8,11 +8,11 @@ import { Breadcrumb, BreadcrumbItem, Container } from "@cloudoperators/juno-ui-c
 
 import WelcomeView from "./WelcomeView"
 
-const AppContent = () => {
+const AppContent = (): JSX.Element => {
   return (
     <>
       <Breadcrumb>
-        <BreadcrumbItem icon="home" label="Template App Home" />
+        <BreadcrumbItem onClick={() => {}} label="Template App Home" />
```

Comment:
Following up from https://github.com/cloudoperators/juno/pull/673#discussion_r1881676818...
Can we keep this original? `icon="home"`
This would default to `help`, if not provided.

Reply:
I will rollback it too good catch Mrs :)